packages/backend/src/routes/AirdropRouter.ts

```diff
@@ -52,6 +52,11 @@ async function getAirdrop(req, res) {
         res.status(422).json({ error: error })
         return
     }
+    if (negRepProof.publicSignals[4] > DEFAULT_AIRDROPPED_KARMA) {
```

Comment:
you can use `negRepProof.negRep` instead of `negRepProof.publicSignals[4]`
`NegRepProof` decode the public signals

Reply:
@kittybest this hasn't been changed